build/NuGetPackages/Microsoft.Build.Conversion.Core.nuspec

```diff
@@ -0,0 +1,26 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
```

Comment:
These `.nuspec`s are going to conflict with the ones @tmeschter introduced in `master` with 6f17735d76a3c9fb11a93795fec98572c35a2b10, aren't they? Can you coordinate on location/content?